features/networking/egress-ingress.feature

```diff
@@ -109,21 +109,18 @@ Feature: Egress-ingress related networking scenarios
     And evaluation of `project.name` is stored in the :proj1 clipboard
  
     # Create egress policy in project-1
-    And evaluation of `BushSlicer::Common::Net.dns_lookup("yahoo.com")` is stored in the :yahoo_ip clipboard
-    When I download a file from "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/egress-ingress/dns-egresspolicy1.json"
-    And I replace lines in "dns-egresspolicy1.json":
-      | 98.138.0.0/16 | <%= cb.yahoo_ip %>/32 |
     And I run the :create admin command with:
-      | f | dns-egresspolicy1.json |
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/egress-ingress/dns-egresspolicy1.json |
       | n | <%= cb.proj1 %> |
     Then the step should succeed
  
     # Check egress rule added in openflow
     Given I select a random node's host
-    When I run the ovs commands on the host:
-       | ovs-ofctl dump-flows br0 -O openflow13 \| grep <%= cb.yahoo_ip %> |
```

Comment:
Above command is executed on hosts and it need to ssh to the host. It can be replaced by `run command on the "<%= cb.node_name %>" node's sdn pod`.

Reply:
It is removed (red lines). Do you see any command on the host that is not removed?

Reply:
No, all ovs commands on the host have been removed.
Sorry for my confusing comments. I means the fix in this PR looks good and we should follow it and update other cases like this, since this PR just updates some critical cases.
Except the `ovs` command there are some other commands on the host in networking cases, if I am not mistaken. I'd like to see the follow up PRs to update them.